onadata/apps/api/viewsets/xform_submission_viewset.py

```diff
@@ -60,6 +82,12 @@ def get_serializer_class(self):
         if 'application/x-www-form-urlencoded' in content_type:
             return RapidProSubmissionSerializer
 
+        if 'application/vnd.org.flowinterop.results+json' in content_type:
```

Comment:
Can we have this in a constant variable, it's being used in a couple of places in this file.

```python
FLOIP_RESULTS_CONTENT_TYPE = 'application/vnd.org.flowinterop.results+json'
```
Then
```python
if FLOIP_RESULTS_CONTENT_TYPE in content_type:
    ....
    request.accepted_media_type = FLOIP_RESULTS_CONTENT_TYPE
```